pkg/catalog/catalog_test.go

```diff
@@ -3,20 +3,21 @@ package catalog_test
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/weaveworks/profiles/api/v1alpha1"
+
+	profilesv1alpha1 "github.com/weaveworks/profiles/api/v1alpha1"
```

Comment:
can we call this `profilev1` similar to ho we have `helmv2` & `sourcev1`?

Reply:
Yeah that sounds good to me. It's shorter to. :) Are you okay with that @bigkevmcd ?